include/nbl/core/math/glslFunctions.tcc

```diff
@@ -473,6 +473,8 @@ NBL_FORCE_INLINE T gcd(const T& a, const T& b)
 	return std::gcd(a,b);
 }
 
+// https://libcxx.llvm.org/docs/Cxx1zStatus.html "Mathematical Special Functions for C++17"
+#ifndef _NBL_ANDROID_API_
```

Comment:
if after we compile Android, and this isnt really implemented, then you need to open an issue asking for an implementation of these.